myadmin-angular/angular.json

```diff
@@ -743,6 +743,242 @@
           }
         }
       }
+    },
+    "official-study-material-app": {
+      "root": "projects/official-study-material-app/",
+      "sourceRoot": "projects/official-study-material-app/src",
+      "projectType": "application",
+      "prefix": "unisa",
+      "schematics": {
+        "@schematics/angular:component": {
+          "styleext": "scss",
+          "spec": false
+        },
+        "@schematics/angular:class": {
+          "spec": false
+        },
+        "@schematics/angular:directive": {
+          "spec": false
+        },
+        "@schematics/angular:guard": {
+          "spec": false
+        },
+        "@schematics/angular:module": {
+          "spec": false
+        },
+        "@schematics/angular:pipe": {
+          "spec": false
+        },
+        "@schematics/angular:service": {
+          "spec": false
+        }
+      },
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/official-study-material-app",
+            "index": "projects/official-study-material-app/src/index.html",
+            "main": "projects/official-study-material-app/src/main.ts",
+            "polyfills": "projects/official-study-material-app/src/polyfills.ts",
+            "tsConfig": "projects/official-study-material-app/tsconfig.app.json",
+            "assets": [
+              "projects/official-study-material-app/src/favicon.ico",
+              "projects/official-study-material-app/src/assets"
+            ],
+            "styles": [
+              "projects/official-study-material-app/src/styles.scss",
+              "node_modules/bootstrap/dist/css/bootstrap.min.css",
+              "node_modules/angular2-toaster/toaster.css",
+              "node_modules/@clr/icons/clr-icons.min.css"
+            ],
+            "scripts": []
+          },
+          "configurations": {
+            "production": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/official-study-material-app/src/environments/environment.ts",
+                  "with": "projects/official-study-material-app/src/environments/environment.prod.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true
```

Comment:
Just a headsup: We should add the baseHref  parameter to the production build configuration, e.g. "baseHref": "/official-study-material-app/".

In my quest to find a solution for passing the CAS authz headers from Apache to Angular I have found that the new version of angular-cli does not work anymore when passing in the base-href command argument.
We need this baseHref parameter to be set when deploying the Angular app to Apache. We (or at least I while testing the apsp in an Apache web server instance on my local machine) are deploying each app as directory in Apache. If we were to copy the app web resources directly into Apache's htdocs directory the base-href value is not necessary.
https://github.com/angular/angular-cli/issues/10736
https://angular.io/guide/deployment